internal/devices/drivers.go

```diff
@@ -0,0 +1,35 @@
+// +build windows
+
+package devices
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Microsoft/hcsshim/internal/log"
+	"github.com/Microsoft/hcsshim/internal/resources"
+	"github.com/Microsoft/hcsshim/internal/uvm"
+)
+
+// InstallWindowsDriver mounts a specified kernel driver using vsmb, then installs it in the UVM
```

Comment:
This comment should also mention:
- The `driver` arg is the path to the driver directory on the host.
- A resource closer for the VSMB share is returned.

Reply:
Also done